test/performance/observed_concurency_test.go

```diff
@@ -64,28 +60,27 @@ func generateTraffic(client *spoof.SpoofingClient, url string, concurrency int,
 				case <-done:
 					return nil
 				default:
-					res, _ := client.Do(req)
+					res, err := client.Do(req)
+					if err != nil {
+						logger.Infof("Error sending request: %v", err)
+					}
 					if resChannel != nil {
```

Comment:
I think we gan cet rid of this check.

Reply:
Not without continuing on the error path :(

Reply:
I think we can get rid of the `if`, we should still post to channel.